wd_helper.au3

```diff
@@ -1297,12 +1252,7 @@ Func _WD_IsLatestRelease()
 		EndIf
 	EndIf
 
-	If $_WD_DEBUG = $_WD_DEBUG_Info Then
-		__WD_ConsoleWrite($sFuncName & ': ' & $bResult)
-	EndIf
-
-	Return SetError(__WD_Error($sFuncName, $iErr), $_WD_HTTPRESULT, $bResult)
-
+	Return SetError(__WD_Error($sFuncName, $iErr, $bResult), 0, $bResult)
```

Comment:
This needs to be `Return SetError(__WD_Error($sFuncName, $iErr, String($bResult)), 0, $bResult)`. Otherwise, the message won't appear when $bResult is False.

Reply:
Fixed